pyfolio/plotting.py

```diff
@@ -1660,3 +1662,126 @@ def plot_prob_profit_trade(round_trips, ax=None):
            xlim=(lower_plot, upper_plot), ylim=(0, y.max() + 1.))
 
     return ax
+
+
+def plot_cone(aggregate_returns, bounds, ax,
+              cone_std=(1, 1.5, 2.),
+              color='green',):
+    """
+    Plots the upper and lower bounds of an n standard deviation
+    cone of forecasted cumulative returns.
+
+    Parameters
+    ----------
+    aggregate_returns : pandas.core.frame.DataFrame
+        Cumulative out-of-sample returns.
+    bounds : pandas.core.frame.DataFrame
+        Contains upper and lower cone boundaries. Column names are
+        strings corresponding to the number of standard devations
+        above (positive) or below (negative) the projected mean
+        cumulative returns.
+    ax : matplotlib.Axes, optional
+        Axes upon which to plot.
+    cone_std : int, float, or list of int/float
```

Comment:
Think this docstring needs to be updated - based on the code below and the default value above, it looks like `cone_std` needs to be an iterable of int/floats. (The for loop will `TypeError` with a single int/float)